information/TASK.md

```diff
@@ -57,6 +52,99 @@ export function model (): tf.LayersModel {
   return model
 ```
 
+#### Preprocessing
+
+In the Task object we can optionally choose to add preprocessing functions. Preprocessing is defined [here](../discojs/src/dataset/preprocess.ts),
+and is currently only implemented for images. The predefined functions (resize, normalize, ...) can be found there, and it is also there
+where you can easily create custom preprocessing.
+
+> Preprocessing is done every time an image is read, this will increase the overhead of training since image loading is lazy; for better performance use data that is already preprocessed.
```

Comment:
i thought it should be batch-wise, for all modalities?

Reply:
I took the pre-existing code for pre processing which was image wise; I'll have a look at how batch wise would look like.

Reply:
Modified it, it is now batch wise